tests/python/integration/load_and_run_tvm_model.py

```diff
@@ -27,8 +27,8 @@ def test_multi_input_multi_output():
     device = 'cpu'
     model = DLRModel(model_path, device)
 
-    assert model._get_output_size_dim(0) == (2, 1)
-    assert model._get_output_size_dim(1) == (3, 1)
+    #assert model._get_output_size_dim(0) == (2, 1)
```

Comment:
why comment out this line?

Reply:
_get_output_size_dim is TVM specific API and not available in new IDLRModel. Perhaps a better 
way is to simply delete these lines.

Reply:
I think we should keep these assertions. We might want to use TVM object directly in this case.

Reply:
You can use `model._impl` instead of `model`
```
    assert model._impl._get_output_size_dim(0) == (2, 1)
    assert model._impl._get_output_size_dim(1) == (3, 1)
```